client/client_test.go

```diff
@@ -224,12 +225,12 @@ func (cs *clientSuite) TestSnapClientIntegration(c *check.C) {
 	defer srv.Close()
 
 	cli := client.New(nil)
-	options := client.SnapCtlOptions{
-		Context: "foo",
-		Args:    []string{"bar", "--baz"},
+	request := hookstate.SnapCtlRequest{
+		ContextID: "foo",
+		Args:      []string{"bar", "--baz"},
 	}
 
-	stdout, stderr, err := cli.RunSnapctl(options)
+	stdout, stderr, err := cli.RunSnapctl(request)
```

Comment:
This should also be received as a pointer, as the other option values in client.